.gitlab-ci.yml

```diff
@@ -0,0 +1,20 @@
+######################################################################################
+# Copyright 2020 Lawrence Livermore National Security, LLC and other CARE developers.
+# See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+######################################################################################
+
+variables:
+   GIT_SUBMODULE_STRATEGY: normal
+
+stages:
+   - build
+
+gpu:
```

Comment:
Would prefer this to be cuda or something more specific then gpu, because someday we might also have amd gpus

Reply:
Good point. I changed the name to cuda.

Reply:
I will leave more specific names for when we actually do more than one build.

Reply:
Thanks!